### PR TITLE
Feature: Added shortcut for opening folders in secondary pane

### LINKED ISSUE #10907: Feature: Shortcut to open selected folder in Secondary Pane
### What feature or improvement do you think would benefit Files?

Adding a shortcut to open a folder in the secondary pane would help make the secondary pane more useful, and accelerate its use without having to go into the context menu and selecting the option manually. Similar to the use of Ctrl + Enter to open a folder in a new tab.

### Requirements

- `ctrl` + `shift` + `enter` should open folder in new pane

### Files Version

2.4.8.0

### Windows Version

Version	10.0.22621 Build 22621

### Comments

_No response_

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #10907 

**What has been done**
- Made the process to open a folder in the dual pane into a method inside `NavigationHelpers`.
- Added the keyboard accelerator to both Grid View and Details View. (Not in Column View).
- Added a null check in `FileList_PreviewKeyDown` for each view.
- Fixed the missing icon on the flyout menu to open in dual pane option. (Used the one from the Settings)

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots**

With icon:
IconFlyoutMenu

Without icon:
image

### CODE REVIEW

**Comment by hecksmosis** [comment]
Aside from that LGTM

**Comment by QuaintMako** [comment]
> Aside from that LGTM and works for me

Aside from? I suppose your review did not get through.

**Comment by hecksmosis** [comment]
Does it not appear?

**Comment by hecksmosis** [comment]
Wouldn't it be better to rename the function to OpenInSecondaryPane for consistency?

File: src/Files.App/Helpers/NavigationHelpers.cs [hunk 1]
```diff
@@ -36,6 +37,19 @@ public static Task<bool> OpenTabInNewWindowAsync(string tabArgs)
...
+				return;
+
+			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
+			if (!userSettingsService.PreferencesSettingsService.IsDualPaneEnabled)
+				return;
+
+			if (listedItem is not null)
+				associatedInstance.PaneHolder?.OpenPathInNewPane((listedItem as ShortcutItem)?.TargetPath ?? listedItem.ItemPath);
+		}
+
```
Comment (ferrariofilippo) [inline src/Files.App/Helpers/NavigationHelpers.cs:52]: Why here `listedItem` is null checked twice?
I think you should also change `IUserSettingsService` to `var`

**Review by ferrariofilippo — CHANGES_REQUESTED** [review]

File: src/Files.App/Helpers/NavigationHelpers.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Helpers/NavigationHelpers.cs:52]: Good catch, looks like it can be removed

File: src/Files.App/Helpers/NavigationHelpers.cs [hunk 2]
```diff
@@ -36,6 +37,19 @@ public static Task<bool> OpenTabInNewWindowAsync(string tabArgs)
 			return Launcher.LaunchUriAsync(folderUri).AsTask();
 		}
 
+		public static void OpenInSecondaryPane(IShellPage associatedInstance, ListedItem listedItem)
+		{
+			if(associatedInstance is null || listedItem is null)
+				return;
+
+			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
```
Comment (yaira2) [inline src/Files.App/Helpers/NavigationHelpers.cs:45]: Can the service be declared at the top of the class?

**Review by ferrariofilippo — APPROVED** [review]
LGTM

**Review by yaira2 — APPROVED** [review]

File: src/Files.App/Helpers/ContextFlyoutItemHelper.cs [hunk 3]
```diff
@@ -688,10 +688,10 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
 				new ContextMenuFlyoutItemViewModel()
 				{
 					Text = "BaseLayoutItemContextFlyoutOpenInNewPane/Text".GetLocalizedResource(),
-					Glyph = "\uE117",
+					Glyph = "\xF117",
```
Comment (d2dyno1) [inline src/Files.App/Helpers/ContextFlyoutItemHelper.cs:691]: What does it change?

Comment (yaira2) [reply src/Files.App/Helpers/ContextFlyoutItemHelper.cs:691]: The previous glyph had a typo so it didn't load

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**